pkg/csharp/Makefile

```
@@ -0,0 +1,25 @@
+all: build
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)